.devcontainer/host_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+# Prepare to install packages
+sudo apt update
+
+# Install the prerequisites
+sudo apt install -y --no-install-recommends wget sudo xz-utils nano bc git python3 bison flex make cppcheck gdb
+
+# Optional: install python tests prerequisites
+sudo apt install -y --no-install-recommends python3-pip python3-setuptools python3-psutil python3-six python3-wheel
+pip3 install timeout_decorator thrift==0.11.0 osquery pexpect==3.3
+
+# Optional: install RPM packaging prerequisites
+sudo apt install -y --no-install-recommends rpm binutils
+
+# Download and install the osquery toolchain
+wget -O /tmp/osquery-toolchain.tar.xz https://github.com/osquery/osquery-toolchain/releases/download/1.1.0/osquery-toolchain-1.1.0-$(uname -m).tar.xz
```

Comment:
Quite possible this should be converted to an environment variable or similar; right now you'd need to come in and update this file then rebuild the Devcontainer

Reply:
Perhaps something like this?

```suggestion
sudo apt install -y jq
version=$(curl https://api.github.com/repos/osquery/osquery-toolchain/releases | jq -r '.[0].tag_name')
wget -O /tmp/osquery-toolchain.tar.xz https://github.com/osquery/osquery-toolchain/releases/download/${version}/osquery-toolchain-${version}-$(uname -m).tar.xz
```

Reply:
Very nice, love pulling latest like this